src/actions/profileActions/profileActions.js

```diff
@@ -0,0 +1,57 @@
+import axios from 'axios';
+import {notify} from 'react-notify-toast';
+import requestLoadingAction from '../authActions/requestLoading';
+import {
+	fetchProfileFailure,
+	fetchProfileSuccess,
+	updateProfileFailure,
+	updateProfileSuccess
+} from './profileActionCreator';
+import { API_URLS, USERNAME_KEY } from '../../constants';
+
+const NotificationSuccess = (message) => notify.show(message, 'success', 5000);
+
+const NotificationError = (message) => notify.show(message, 'error', 5000);
+
+const myHeader = {headers: {'Content-Type': 'application/json', 'Accept': 'application/json',
+	'Authorization': `Token ${ localStorage.getItem('auth_token')}`}};
+export const fetchProfile = (props) => (dispatch) => {
```

Comment:
format the code here to improve readability. 

Blank lines can come in handy

Reply:
We handled this.